src/man/sss-certmap.5.xml

```diff
@@ -35,6 +35,13 @@
             searched in the userCertificate attribute as DER encoded binary. If
             no domains are given only the local domain will be searched.
```

Comment:
What is "local domain" here?

Reply:
The domain the client is in, i.e. the domain the client is joined to.